tools/importer-msgraph-metadata/cmd/import.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/pandora/tools/importer-msgraph-metadata/pipeline"
+	"github.com/mitchellh/cli"
+)
+
+var _ cli.Command = ImportCommand{}
+
+func NewImportCommand(metadataDirectory, openApiFilePattern, terraformDefinitionsPath, outputDirectory string) func() (cli.Command, error) {
+	return func() (cli.Command, error) {
+		return ImportCommand{
+			metadataDirectory:        metadataDirectory,
+			openApiFilePattern:       openApiFilePattern,
+			outputDirectory:          outputDirectory,
+			terraformDefinitionsPath: terraformDefinitionsPath,
+		}, nil
+	}
+}
+
+type ImportCommand struct {
+	metadataDirectory        string
+	openApiFilePattern       string
+	outputDirectory          string
+	terraformDefinitionsPath string
+}
+
+func (ImportCommand) Synopsis() string {
+	return "Imports Microsoft Graph API metadata and outputs HCL data definitions"
+}
+
+func (ImportCommand) Help() string {
+	return fmt.Sprintf(`Imports and parses Microsoft Graph API metadata
+Arguments:
+
+-tags=applications,servicePrincipals
+        Specifies a comma-separated list of services to import, rather than the entire API.
+
+-versions=v1.0,beta
+        Specified the API versions to import. Defaults to 'v1.0,beta'
+`)
+}
+
+func (c ImportCommand) Run(args []string) int {
+	var apiVersionsRaw, tagsRaw string
+
+	f := flag.NewFlagSet("importer-msgraph", flag.ExitOnError)
+	f.StringVar(&tagsRaw, "tags", "", "A list of comma separated tags to import")
+	f.StringVar(&apiVersionsRaw, "versions", "", "A list of comma separated API versions to import")
+	f.Parse(args)
+
+	apiVersions := []string{"v1.0", "beta"}
+	if apiVersionsRaw != "" {
+		apiVersions = strings.Split(apiVersionsRaw, ",")
+	}
+
+	var tags []string
```

Comment:
Could we default this to a couple of tags for now until we've got this piped the way through? Whilst there's no harm importing all the data, doing so when it's not used will increase the build time etc, so it'd be good to limit this (and increase the GHA size) if we can, until we need all of this data?